style.css

```diff
@@ -0,0 +1,83 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: content-box;
+}
+
+#add-prod-div {
+    margin: 10vh auto;
+    width: 450px
+}
+
+#add-prod-div h2 {
+    margin: 6px auto;
+    text-align: center;
+    color: #0ba589;
+    font-family: 'Poppins', sans-serif;
```

Comment:
Please be committed with the original font family of the design,
Here it is : 'Montserrat', sens-serif.

Reply:
Ok , 
so you mean this is the agreed upon font